calico/felix/fetcd.py

```diff
@@ -442,6 +626,26 @@ def on_workload_delete(self, response, hostname, orchestrator,
             del self.endpoint_ids_per_host[hostname]
 
 
+def _reconnect(etcd_communicator, copy_cluster_id=True):
+    """ Parameter etcd_communicator can be either etcd_watcher or etcd_api"""
+    etcd_addr = etcd_communicator._config.ETCD_ADDR
+    if ":" in etcd_addr:
+        host, port = etcd_addr.split(":")
+        port = int(port)
+    else:
+        host = etcd_addr
+        port = 4001
+    if etcd_communicator.client and copy_cluster_id:
+        old_cluster_id = etcd_communicator.client.expected_cluster_id
+        _log.info("(Re)connecting to etcd. Old etcd cluster ID was %s.",
+                  old_cluster_id)
+    else:
+        _log.info("(Re)connecting to etcd. No previous cluster ID.")
+        old_cluster_id = None
+    etcd_communicator.client = etcd.Client(host=host, port=port,
+                              expected_cluster_id=old_cluster_id)
```

Comment:
I'm nitpicking, but please could you tidy up the indentation of this line? Ideally the second line of arguments should line up with the parens on the first line.